Lib/test/test_functools.py

```diff
@@ -581,6 +581,17 @@ def add(self, x, y):
         for func in [self.A.static, self.A.cls, self.A.over_partial, self.A.nested, self.A.both]:
             self.assertFalse(getattr(func, '__isabstractmethod__', False))
 
+    def test_positional_only(self):
+        def f(a, b, /):
+            return a + b
+
+        p = functools.partial(f, 1)
+        self.assertEqual(p(2), f(1, 2))
+
+        # TODO: Should we make this work?
```

Comment:
I don't think this should work with a keyword - it wouldn't work for current positional-only arguments implemented in the C, I don't think there's an easy way to *make* it work for those.

I do think it would be very useful if `partial` could curry positional arguments other than the first one, though. There are a lot of situations where that could come in handy. Maybe this is good impetus for trying to solve that problem?